### DISCUSSION TITLE
Configurazione iniziale per progetto automazione domestica.

### DISCUSSION THREAD

**Discussion by Flatguy** [indomus/forum#3058] [GENERALE - Discussioni generiche]
Ciao a tutti,
sono Federico e seguo da diversi mesi la chat su Telegram di Indomus.
Purtroppo ho una formazione professionale e scolastica che non mi permette di essere completamente autonomo nel mio progetto in quanto non ho conoscenze nè di elettronica nè di programmazione.
Ma ho intenzione di perseguire nel mio intento di creare una piccola rete domotica a casa mia imperniata soprattutto sul controllo dei consumi, gestione allarme (già esistente) e qualche scenario.
Dopo avere letto e consultato varie guide, ho acquistato per iniziare il SONOFF Zigbee USB dongle più 3.0 SMA da collegare al mio Raspberry Pi5. Ora però, visto che mi mancano le basi tecniche, non so quali altri dispositivi acquistare. 
Come dicevo, mi piacerebbe iniziare con il controllo dei consumi del mio appartamento e la gestione della bascula del mio garage che si trova 3 piani sotto dove vivo io per poi passare alla integrazione dell'allarme Paradox.
Chi riesce gentilmente a darmi un valido supporto in questo progetto?

Grazie a chi mi vorrà rispondere e aiutare.

Federico

**Comment by Bergoglio** [comment]
Legna verde in vista.... :D 
Ovviamente per sdrammatizzare !
Ce la si può fare, non ti preoccupare.

Iniziamo dalla "centrale" : hai già installato Home Assistant sul Raspberry ?
Secondo me era meglio un mini pc, ma poco importa. Come mai hai scelto il raspberry ?

**Reply by Flatguy** [reply]
Ahahahahahah hai ragione!!!
Ho scelto il Raspberry perché ne volevo conoscere le potenzialità, tant'è vero che Home Assistant è venuto dopo.
Si, Home Assistant è già installato e ora mi piacerebbe capire come aggiungere i vari sensori e soprattutto controllare i consumi.
Grazie per la tua risposta

**Comment by Bergoglio** [comment]
Ok, secondo passo : il Sonoff Zigbee dongle è installato e configurato ?
Hai già dei sensori Zigbee ? Perché hai scelto questo standard ?

**Reply by Flatguy** [reply]
Si, installato e configurato e riconosciuto da HA.
Non ho attualmente acquistato alcun sensore o altro Zigbee perché mi vorrei basare sulle esperienze e configurazioni degli altri utenti con molta più esperienza.
Ho scelto Zigbee perché me lo hanno consigliato, in quanto WiFi ha dei limiti così come altri standard.

**Reply by KC-inDomus** [reply]
anche zigbee ha limiti, tanto per cominciare ha un single point of failure in più del wifi.

**Reply by Bergoglio** [reply]
Concordo con KC-inDomus.

Io, ad esempio, ho scelto wi-fi su rete lan e wi-fi di tipo Lorawan.
E, comunque, cerco di collegare il più possibile via cavo (sempre rete lan ma su cavo e non via wi-fi).

**Reply by KC-inDomus** [reply]
lorawan?

**Reply by Flatguy** [reply]
Scusami ma non ho la minima idea di quello che tu stia dicendo

**Reply by snakuzzo** [reply]
https://lora-alliance.org/about-lorawan/

**Comment by KC-inDomus** [comment]
> https://lora-alliance.org/about-lorawan/

cos'è lora lo so bene, mi chiedo cosa c'entri parlando di domotica personale :D

**Reply by snakuzzo** [reply]
non rispondevo a te...ma a @Flatguy  :D

**Reply by KC-inDomus** [reply]
vabbe la mia domanda rimane valida, che c'azzecca? :D

**Reply by Bergoglio** [reply]
Sensori vari tipo questi https://www.milesight.com/iot/#wt-series

**Reply by Bergoglio** [reply]
Mi hanno risolto alcuni problemi, tra cui il segnale che non mi passa attraverso i muri : pensa che non mi arriva il wi-fi tra 2 stanze sovrapposte e collegate direttamente da scala interna...... ed uso tutti apparati lan della Tp-Link serie Omada (per cui sto disseminando casa di access point wifi in mesh).

**Reply by KC-inDomus** [reply]
sì per carità, non discuto lora, è un ottimo standard - ma a mio modo di vedere decisamente sovrabbondante per le necessità domestiche. zigbee, thread e wifi, usati in mesh, sono più che adatti imho.

**Reply by Bergoglio** [reply]
Fosse stato per me avrei fatto tutto in wifi classica, ma a parte del segnale non riuscivo a trovare alcuni sensori come volevo io.
Certo è che Lorawan costa perché trovi solo pezzi "blindati".

Tornando sul pratico, visto che Flatguy è lì con le forbici da elettricista in mano e noi ci dilunghiamo in discussioni filosofiche :D , il mio pensiero è che eviterei troppi standard nello stesso impianto.
Decidi tu Flatguy, secondo me potresti riuscire a fare tutto anche con la semplice lan/wifi e sopperire con Zigbee dove non trovi i componenti adeguati.

**Reply by Flatguy** [reply]
Grazie @Bergoglio, stavo giusto pensando a come farla finita dopo avere letto i vostri messaggi. 
Ad ogni modo, avrei bisogno di tanti ma tanti consigli...
In primis, vorrei tenere monitorati i consumi elettrici delle varie stanze del mio appartamento, in primis cucina e lavanderia.
Come posso fare per monitorare lavatrice, asciugatrice e forno? Anche per evitare distacchi dalla rete.
Grazie.

**Reply by Bergoglio** [reply]
Vuoi monitorare i consumi delle stanze o dei singoli elettrodomestici ?
Partiamo dall'inizio : il software (la domotica) non ti risolve un problema hardware (la potenza disponibile al contatore), al più gli mette una pezza.
Se vuoi evitare i distacchi la domotica può solo non farti partire insieme gli elettrodomestici e quindi ti automatizza quello che già dovresti fare oggi a mano (non farli partire insieme). Per una casa più confortevole dovresti invece, secondo me, in primis valutare l'aumento di potenza al contatore.

**Reply by Flatguy** [reply]
La potenza l'ho già aumentata, ma la moglie usa la casa come Cape Canaveral quindi il distacco è sempre dietro l'angolo. 
Mi piacerebbe monitorare le prese a cui sono attaccate le utenze più critiche, anche i climatizzatori. 
Grazie per la pazienza

**Comment by KC-inDomus** [comment]
> Grazie @Bergoglio, stavo giusto pensando a come farla finita dopo avere letto i vostri messaggi. Ad ogni modo, avrei bisogno di tanti ma tanti consigli... In primis, vorrei tenere monitorati i consumi elettrici delle varie stanze del mio appartamento, in primis cucina e lavanderia. Come posso fare per monitorare lavatrice, asciugatrice e forno? Anche per evitare distacchi dalla rete. Grazie.

dipende come hai organizzato l'impianto elettrico, in primis. senza questa informazione è impossibile darti consigli sensati 👍

**Comment by Bergoglio** [comment]
Ha ragione KC, comunque la soluzione più semplice è di collegare le spine di questi elettrodomestici frapponendo delle spine domotiche.
Bisogna controllare solo che le spine domotiche che compri sopportino il carico massimo dell'elettrodomestico.
Un dubbio comunque che mi viene in mente : lavatrice, asciugatrice, etc. sicuramente hanno la spina ma il forno ? Spina o cavi collegati a morsetti ? I climatizzatori quasi sicuramente non hanno le spine  ma cavi su morsetti e generalmente prendono tensione dall'unità esterna.
Inoltre, il forno : ammettiamo che sia collegato con spina, c'è lo spazio per inserire una spina domotica ?

**Reply by Flatguy** [reply]
Se non sbaglio ci sono gli Shelly che "dovrebbero" fare quello che sto cercando io.
In caso negativo, chiedo scusa.
Purtroppo non ho competenze elettriche tali da montare anche una lampadina (nella vita faccio altro).

**Reply by Bergoglio** [reply]
Esatto, ci sono le spine della Shelly e sono wifi (LAN).
Bisogna solo verificare che supportano il carico del forno / lavatrice / etc. (mi pare di sì)
Questo riesci a verificarlo tu ?
Domanda : ma la spina del forno la monteresti tu ?
